### PR TITLE
Research layout

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
* Moved the related colleges up into a column next to the main content.
* Added the post-list layout for people.
* Updated the layout for publications. We're likely going to change the layout for projects.

**Motivation and Context**
Making some common sense adjustments to the layout before we spend some time actually formalizing how things are going to look. This should make it a little easier to adjust things for some of these components going forward as well.

**How Has This Been Tested?**
Check out the cyber security page in dev.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: includes/person-functions.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * @param array $faculty An array of faculty posts
+ * @return string
+ */
+function research_get_faculty_list( $faculty ) {
+	$args = array(
+		'list_title'     => '',
+		'display_search' => false
+	);
+
+	return UCF_Post_List_Common::display_post_list( $faculty, 'people', $args );
```
Comment (cjg89) [inline includes/person-functions.php:19]: We'll need to make sure that the dependency on the post list sc plugin is documented somewhere.

File: includes/person-functions.php [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Person functions
+ */
+
```
Comment (cjg89) [inline includes/person-functions.php:5]: Do we want to use namespacing for this theme vs using function prefixes?

File: includes/person-functions.php (hunk 1 above)
Comment (jmbarne3) [reply includes/person-functions.php:19]: I'll add an issue to make sure this gets added to the wiki along with ACF.

File: includes/person-functions.php (hunk 2 above)
Comment (jmbarne3) [reply includes/person-functions.php:5]: For some reason I didn't start with namespacing on this project. If we want it, I can add it in throughout the project in a future PR.

**Pull request merged**